### DISCUSSION TITLE
Prevent to open scrollback in case Neovim is open

### DISCUSSION THREAD

**Discussion by igorgue** [mikesmithgh/kitty-scrollback.nvim#275] [Q&A]
I get very confused when I'm running neovim and press "shift+ctrl+h" and this plugin opens on top of neovim itself, looking no different...

So I wonder I can disable it from running if neovim itself is running.

**Comment by mikesmithgh** [comment]
Hi @igorgue, thanks for the comment. Not at the moment. I actually use this if for example there is some text in Neovim that I can't select, maybe a command I am currently typing, or text in fzf-lua/telescope, or other edge cases. 

There is logic in place to prevent you from opening kitty-scrollback.nvim if you have already opened kitty-scrollback.nvim. I may be able to hook into some similar logic. I'll open a feature request.

**Reply by mikesmithgh** [reply]
opened https://github.com/mikesmithgh/kitty-scrollback.nvim/issues/277
